inc/vcf/parsing_state.hpp

```diff
@@ -43,7 +45,8 @@ namespace ebi
         std::shared_ptr<std::vector<Record>> records;
         std::shared_ptr<std::vector<std::unique_ptr<Error>>> errors;
         std::shared_ptr<std::vector<std::unique_ptr<Error>>> warnings;
-     
+        RecordCache duplicates;
```

Comment:
Mmmh, no, it's not used. It helped me to design the `normalize` interface, I will remove it.